Zilon.Core/Zilon.Bot.Players/Logics/ExitLogicState.cs

```diff
@@ -10,45 +10,35 @@ namespace Zilon.Bot.Players.Logics
 {
     public sealed class ExitLogicState : LogicStateBase
     {
-        private readonly ISector _sector;
-        private readonly ISectorMap _map;
-
         private MoveTask _moveTask;
 
-        public ExitLogicState(ISectorManager sectorManager)
+        public override IActorTask GetTask(IActor actor, ISectorTaskSourceContext context, ILogicStrategyData strategyData)
         {
-            if (sectorManager is null)
-            {
-                throw new System.ArgumentNullException(nameof(sectorManager));
-            }
+            var sector = context.Sector;
+            var map = sector.Map;
 
-            _sector = sectorManager.CurrentSector;
-            _map = sectorManager.CurrentSector.Map;
-        }
-
-        public override IActorTask GetTask(IActor actor, ILogicStrategyData strategyData)
-        {
             if (!strategyData.ExitNodes.Any())
             {
                 Complete = true;
                 return null;
             }
 
             var actorNode = actor.Node;
-            if (_map.Transitions.TryGetValue(actorNode, out var currentTransition))
+            if (map.Transitions.TryGetValue(actorNode, out var currentTransition))
             {
-                _sector.UseTransition(currentTransition);
+                System.Console.WriteLine("trans");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)